schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -483,6 +483,8 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_SRV" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_TXT" },
                   { "$ref": "https://schema.management.azure.com/schemas/2016-03-01/Microsoft.Insights.json#/resourceDefinitions/alertrules" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2017-06-01-preview/Microsoft.AzureStack.json#/resourceDefinitions/registrations" },
```

Comment:
>2017-06-01-preview [](start = 73, length = 18)

2017-06-01?

Reply:
Yup.

Reply:
This is not fixed. 

---
In reply to: [168658962](https://github.com/Azure/azure-resource-manager-schemas/pull/413#discussion_r168658962) [](ancestors = 168658962)

Reply:
You did not tell me what was wrong. Just asked me 2017-16-01?  What does that even mean.

Reply:
I figured out what you attempted to say.  Removed preview part of version.